Core/HLE/sceFont.cpp

```diff
@@ -98,6 +98,10 @@ struct FontRegistryEntry {
 static const FontRegistryEntry fontRegistry[] = {
 	// This was added for Chinese translations and is not normally loaded on a PSP.
 	{ 0x288, 0x288, 0x2000, 0x2000, 0, 0, FONT_FAMILY_SANS_SERIF, FONT_STYLE_DB, 0, FONT_LANGUAGE_CHINESE, 0, 1, "zh_gb.pgf", "FTT-NewRodin Pro DB", 0, 0, 1581700, 145844, true },
+	
+	// This is also added for Chinese translations and is not normally loaded on a PSP.	
+	{ 0x288, 0x288, 0x2000, 0x2000, 0, 0, FONT_FAMILY_SANS_SERIF, FONT_STYLE_DB, 0, FONT_LANGUAGE_CHINESE, 0, 1, "chn.pgf", "FTT-NewRodin Pro DB", 0, 0, 1581700, 145844 },
```

Comment:
I'm a little confused. Is there any difference "zh_gb.pgf" and "chn.pgf"?

Reply:
chn.pgf use older ppsspp version 's jpn0.pgf ( v1.10.3-206-g9f88a8835)